### PR TITLE
Daniel: finished 09

### PR DESCRIPTION
**Pull request by dk1code**

### CODE REVIEW

File: 09-PB-language-number/daniel.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// 1.
+let stringT1 = "Hello, my name is Daniel and I am ";
+let ageT1 = 35;
+let concatenateT1 = stringT1 + ageT1;
+console.log(`1. ${concatenateT1}`);
+
+// 2.
+
+let stringInt = "1005", stringFloat = "10.05";
+console.log(`2.\nInteger: ${parseInt(stringInt)}\nFloat number: ${parseFloat(stringFloat)}`);
```
Comment (martl) [inline 09-PB-language-number/daniel.js:12]: if you are using the backticks you don't actually need to escape the new line, I would just put a REAL new line instead.

File: 09-PB-language-number/daniel.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+// 1.
+let stringT1 = "Hello, my name is Daniel and I am ";
+let ageT1 = 35;
+let concatenateT1 = stringT1 + ageT1;
```
Comment (Ungoot) [inline 09-PB-language-number/daniel.js:6]: Exercise asks;
```
In this case, what is the + operator doing? Comment your answer in the js file.

File: 09-PB-language-number/daniel.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+console.log(`2.\nInteger: ${parseInt(stringInt)}\nFloat number: ${parseFloat(stringFloat)}`);
+
+// 3.
+
+let a = 30, b = 939, c = 40.9;
+let moduloA = a % 2;
+let moduloB = b % 2;
+let moduloC = c % 2;
+console.log(
+  `3.\n${a} is ${(moduloA = 0
```
Comment (martl) [inline 09-PB-language-number/daniel.js:21]: same here

Comment (martl) [reply 09-PB-language-number/daniel.js:21]: instead of = 0 you need to say === 0

File: 09-PB-language-number/daniel.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    ? "an even number"
+    : "an odd number")}\n${b} is ${(moduloB = 0
+    ? "an even number"
```
Comment (martl) [inline 09-PB-language-number/daniel.js:24]: same here

File: 09-PB-language-number/daniel.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    : "an odd number")}\n${c} is ${(moduloC = 0
+    ? "an even number"
```
Comment (martl) [inline 09-PB-language-number/daniel.js:26]: and here

File: 09-PB-language-number/daniel.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    : "an odd number")}`
+);
```
Comment (Ungoot) [inline 09-PB-language-number/daniel.js:28]: This returns 939 and 40.9 as odd numbers.

**Pull request closed without merging**